src/dmd/frontend.h

```diff
@@ -14187,18 +14190,30 @@ extern void verror(const Loc& loc, const char* format, va_list ap, const char* p
 
 extern void verrorSupplemental(const Loc& loc, const char* format, va_list ap);
 
+extern void _verrorSupplemental(const Loc& loc, const char* format, va_list ap);
```

Comment:
BTW do we really need to expose these wrapper functions to the outside? Shouldn't they be private and extern(D)?

Reply:
Not necessarily, seems like it's probably better to remove them from the C++ interface-

Reply:
Done